docs/limitations.md

```diff
@@ -1,12 +1,30 @@
 # Current limitations
 
-1. Does not work on Windows. Tested on Mac and Linux only. Tracking issue: https://github.com/prisma/prisma2/issues/4
-2. Processes 1 request at a time, no parallelism during the early preview period. This will be fixed before GA. 
-3. Some edge cases for complex nested mutations don't work properly. 
-4. Limited auto-complete in Typescript projects due to a compiler bug. We raised a PR to fix that [here](https://github.com/microsoft/TypeScript/pull/32100). 
-5. No realtime API/subscriptions. In the future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet. 
-6. Models must have an `@id` attribute and it must take one of these forms:
+During the preview phase of Prisma 2 there are missing features, limited performance and stability issues you should be aware of:
```

Comment:
Nitpick: We've typically uppercased "Preview". Rationale: "General Availability" is typically also uppercased. It's basically a _name_ for a certain phase of the software lifecycle.

Reply:
I should remove the "phase" then, right?

Reply:
Yup I think that sounds a bit better. But I think it's generally fine to speak of the "Preview period" or "Preview phase".

Reply:
"Preview period" is best, changed it.